### ISSUE TITLE
detect rule instead of string

### ISSUE THREAD

**Issue by rekby** [go-lang-plugin-org/go-lang-idea-plugin#1167]
code:
```go
func f1(asd string) {

}

func f2() {
	a := "asd"
	f1(a[1:])
}
```

editor show error on line `f1(a[1:])`:
`cannot use a[1:] (type []rune) as type string in argument to f1.`

But it is string, not rule and the example copmile and work fine.

**Comment by mtoader** [comment]
What is the plugin version that you are using ?

**Comment by rekby** [comment]
0.9.16-dev-0.9.16-alpha.9

**Comment by dlsniper** [comment]
Fixed in 1.0
